zh-translation:/istio.io/content/zh/docs/tasks/observability/distributed-tracing/zipkin

charili opened the pull request:
ref: https://github.com/servicemesher/istio-official-translation/issues/1306

istio-testing commented:
Hi @charili. Thanks for your PR.

I'm waiting for a [istio](https://github.com/orgs/istio/people) member to verify that this patch is reasonable to test. If it is, they should reply with `/ok-to-test` on its own line. Until that is done, I will not automatically test new commits in this PR, but the usual testing commands by org members will still work. Regular contributors should [join the org](https://github.com/orgs/istio/people) to skip this step.

Once the patch is verified, the new status will be reflected by the `ok-to-test` label.

I understand the commands that are listed [here](https://go.k8s.io/bot-commands?repo=istio%2Fistio.io).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

geeknoid commented:
/ok-to-test

istio-testing commented:
@charili: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
lint_istio.io | f5136d43367f2780a2cfef9391cb16fb9d2fd016 | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_istio.io/5542/lint_istio.io/1593) | `/test lint_istio.io`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@charili: PR needs rebase.

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

istio-testing commented:
@charili: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
lint_istio.io | 02499a98b9b8bdaac5a6cd45ecf3862e85fbb2ec | [link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_istio.io/5542/lint_istio.io/1615) | `/test lint_istio.io`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

charili commented:
@googlebot I fixed it

googlebot <notifications@github.com> 于2019年11月9日周六 下午8:05写道：

> We found a Contributor License Agreement for you (the sender of this pull
> request), but were unable to find agreements for all the commit author(s)
> or Co-authors. If you authored these, maybe you used a different email
> address in the git commits than was used to sign the CLA (login here
> <https://cla.developers.google.com/> to double check)? If these were
> authored by someone else, then they will need to sign a CLA as well, and
> confirm that they're okay with these being contributed to Google.
> In order to pass this check, please resolve this problem and then comment @googlebot
> I fixed it.. If the bot doesn't comment, it means it doesn't think
> anything has changed.
>
> ℹ️ *Googlers: Go here
> <https://goto.google.com/prinfo/https%3A%2F%2Fgithub.com%2Fistio%2Fistio.io%2Fpull%2F5542>
> for more info*.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/istio/istio.io/pull/5542?email_source=notifications&email_token=ABXL754D2ITGB5MYJEPKJP3QS2RQXA5CNFSM4JLGFPA2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEDUEPWQ#issuecomment-552093658>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABXL752Y4F42VFO5FNNTGD3QS2RQXANCNFSM4JLGFPAQ>
> .
>

tanjunchen commented:
@charili  冲突太多 建议关闭

charili commented:
> @charili 冲突太多 建议关闭

@tanjunchen 关闭后重新提交吗？

tanjunchen commented:
@charili  目前这个没有被翻译，我的建议是：
把你翻译的内容拷贝出来，重新 reset  --hard 上游分支 再翻译相关文件就行
重新提交到你的这个分支 -f
